.idea/compiler.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
All files inside idea folder should be removed from repo and add to gitignore properly

Reply:
it was added after the creation of new branch